tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/functions/FunctionSignatureTest.java

```diff
@@ -444,4 +444,30 @@ public static ArrayValue mockedNativeFuncWithOptionalParams(long a, double b, St
         tuple.add(4, (Object) e);
         return tuple;
     }
+
+    @Test(description = "Test1: function signature which has a function typed param with only rest param")
```

Comment:
Since recently, we've been trying to write the tests more in Ballerina. For example, https://github.com/ballerina-platform/ballerina-lang/blob/133d9010951cf306ffc708243a18bc3e9f111c22/tests/jballerina-unit-test/src/test/resources/test-src/expressions/mappingconstructor/spread_op_field.bal#L31.
Shall we try and do the same for the newly added tests?

Reply:
Thanks. Changed made